docs/examples/basecoin/cmd/basecli/main.go

```diff
@@ -80,7 +80,7 @@ func main() {
 			stakecmd.GetCmdQueryRedelegation("stake", cdc),
 			stakecmd.GetCmdQueryRedelegations("stake", cdc),
 			slashingcmd.GetCmdQuerySigningInfo("slashing", cdc),
-			authcmd.GetAccountCmd("acc", cdc, types.GetAccountDecoder(cdc)),
+			authcmd.GetAccountCmd("acc", cdc),
```

Comment:
Should `"acc"` be a constant somewhere (`storeAcc` maybe)?

Reply:
I don't know really. This is how I found the code. I've opened [this issue](https://github.com/cosmos/cosmos-sdk/issues/2838) to track. I'll update to make it a `const`.